test/api_integration/apis/saved_objects/find.ts

```diff
@@ -256,6 +258,7 @@ export default function ({ getService }) {
             )
             .expect(400)
             .then((resp) => {
+              // eslint-disable-next-line no-console
```

Comment:
Do we really need this `console.log`? For what purpose? @rudolf as one of the logs added in https://github.com/elastic/kibana/pull/44143

Reply:
The log line I added was just for debugging so it shouldn't have been commited.

Reply:
I'll remove them then. Thanks for confirming!